storage/Filesystems/Btrfs.cc

```diff
@@ -55,6 +70,48 @@ namespace storage
     }
 
 
+    BtrfsSubvolume*
+    Btrfs::get_top_level_btrfs_subvolume()
+    {
+	return get_impl().get_top_level_btrfs_subvolume();
+    }
+
+
+    const BtrfsSubvolume*
+    Btrfs::get_top_level_btrfs_subvolume() const
+    {
+	return get_impl().get_top_level_btrfs_subvolume();
+    }
+
+
+    vector<BtrfsSubvolume*>
+    Btrfs::get_btrfs_subvolumes()
```

Comment:
No, there is const and non-const version. It like operator[] for std::vector (http://en.cppreference.com/w/cpp/container/vector/operator_at).